--- a/src/wizard_funcs.cxx.old	2012-10-19 22:59:36.248454183 +0200
+++ b/src/wizard_funcs.cxx	2012-10-19 22:59:58.643118127 +0200
@@ -772,7 +772,7 @@ Wizard::preview_aircraft()
         }
         catch (const sg_exception& exc )
         {
-            fl_alert( exc.getFormattedMessage().c_str() );
+            fl_alert( "%s", exc.getFormattedMessage().c_str() );
             return;
         }
         catch (...)
